code/modules/projectiles/projectile/bullets/smg.dm

```diff
@@ -50,3 +50,22 @@
 	name = "4.6x30mm rubber bullet"
 	damage = 5
 	stamina = 22
+
+/obj/item/projectile/bullet/c46x30mm/venom
+	name = "4.6x30mm venom bullet" // Doesnt actually inject toxin. Just does toxin damage.
+	damage = 6
+
+/obj/item/projectile/bullet/c46x30mm/venom/on_hit(atom/target, blocked)
+	if((blocked != 100) && iscarbon(target))
+		var/mob/living/carbon/victim = target
+		victim.reagents.add_reagent(/datum/reagent/toxin, 6)
```

Comment:
this is actually just injecting toxin though?

Reply:
i forgot to update the documentation 💀